app/css/Mindfulness.css

```diff
@@ -1,28 +1,45 @@
+
 /* Mindfulness.css */
-.video-container {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
+/* Set a responsive base font size */
+html {
+  font-size: calc(1rem + 0.5vw); /* Base font size adjusts according to screen size and system settings */
+}
+
+body {
+  font-size: 1rem; /* Base font size for body text */
+}
+
+h1 {
+  font-size: 2rem; /* Scales with the base font size */
 }
 
-.video-wrapper {
-  position: relative;
-  width: 100%;
-  height: 0;
-  padding-bottom: 56.25%; /* 16:9 aspect ratio */
+h2 {
+  font-size: 1.75rem; /* Scales with the base font size */
 }
 
-.video {
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
+p {
+  font-size: 1rem; /* Same as body font size */
+}
+
+button {
+  font-size: 1rem; /* Scales with the base font size */
 }
 
 .video-title {
-  font-size: 20px;
-  display: inline-block;
```

Comment:
Why removing display style?

Reply:
Flexbox is powerful in handling the alignment and layout of its children. With flexbox, we can center, align, and distribute space among the child elements without needing to specify display: inline-block;. The flex container manages these aspects, making inline-block redundant.